File: #Task-03/Js/Sub-Task 4/Sub-Task 4.js
const fs=require("fs");
 function write(n){
    if ((n%2)==1){
        x=Math.floor(n/2);
        var c=n-x-1;
        var m=1;
        var s1=-1;
        var s2=1;
        var spa;
        var s;
        for(var g=0;g<n;g++){
            if (c!=0){
                spa=(" ".repeat(c));
            }
            else{
                spa="";
            }
            s=spa+"*".repeat(m)+"\n";
            fs.appendFile("output.txt",s,err=>{
                if(err){
                    console.err;
                    return;
                }
            })
            m+=(s2*2)
            c+=s1
            if (c==0){
                s1=1}
            if (m>n){
                s2=-1
                m=n-2}
        }

    }else{
        console.log("Pls Enter A Even Number");
    }       
}
fs.readFile("input.txt","utf-8",(err,data)=>{
    fs.writeFile("output.txt","",err=>{
        if(err){
            console.err;
            return;
        }
    })
    write(data);
    console.log("Finished")
});

File: #Task-03/Java/Sub-Task 4/SubTask4.java
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileWriter;
import java.io.IOException;
import java.util.Scanner;
public class SubTask4{
    public static void main(String[] args) {
        int n=3;
        try{
            File inp=new File("input.txt");
            Scanner myReader=new Scanner(inp);
            while (myReader.hasNextLine()) {
                String data = myReader.nextLine();
                n=Integer.parseInt(data);
            }}catch (FileNotFoundException e){
                System.out.println("An error occurred.");
                e.printStackTrace();
            }
        if (n%2!=0){
            String outputTxt="";
            int c=n-(n/2)-1;
            int m=1;
            int s1=-1;
            int s2=1;
            for (int g=0;g<n;g++){
                if (c!= 0){
                    for (int j=0;j<c;j++){
                        outputTxt+=(" ");
                    }
                }
                for (int j=0;j<m;j++){
                    outputTxt+=("*");
                }
                outputTxt+=("\n");
                m+=s2*2;
                c+=s1;
                if(c==0){
                    s1=1;
                }
                if(m>n){
                    s2=-1;
                    m=n-2;
                }
            }
            try{
                FileWriter ou = new FileWriter("output.txt");
                ou.write(outputTxt);
                System.out.println("Output is Written");
                ou.close();
            }catch(IOException e){
                System.out.println("Error Occured");    
            }
        
        }else{
            System.out.println("Enter odd");
        }
    }
}
